app/controllers/search_controller.rb

```diff
@@ -7,6 +7,20 @@ def show
     # replace non-ASCII characters with ASCII and non-word characters with spaces for API call
     sanitized_search = I18n.transliterate(params[:q]).gsub(/(\W)/, " ")
     @artist = Search.for(sanitized_search).first
-    @related_artists = @artist.related_artists.sort_by {|artist| artist.followers["total"]}.reverse
+    if !@artist
+      begin
+        raise SearchError
+      rescue
+        flash.now[:error] = "Sorry, we couldn't find that artist. Please try again."
+      end
+    else
+      @related_artists = @artist.related_artists.sort_by {|artist| artist.followers["total"]}.reverse
```

Comment:
Line is too long. [101/80]<br>Space between { and | missing.<br>Space missing inside }.